x-pack/plugins/transform/common/privilege/has_privilege_factory.ts

```diff
@@ -21,6 +23,19 @@ export interface Capabilities {
   canUseTransformAlerts: boolean;
   canResetTransform: boolean;
 }
+type Capabilities = { [k in keyof TransformCapabilities]: boolean };
+
+export const INITIAL_CAPABILITIES: Capabilities = {
+  canGetTransform: false,
+  canDeleteTransform: false,
+  canPreviewTransform: false,
+  canCreateTransform: false,
+  canScheduleNowTransform: false,
+  canStartStopTransform: false,
+  canCreateTransformAlerts: false,
+  canUseTransformAlerts: false,
+  canResetTransform: false,
+} as const;
```

Comment:
To make sure nobody mutates this object by accident
```suggestion
export const INITIAL_CAPABILITIES: Capabilities = Object.freeze({
  canGetTransform: false,
  canDeleteTransform: false,
  canPreviewTransform: false,
  canCreateTransform: false,
  canScheduleNowTransform: false,
  canStartStopTransform: false,
  canCreateTransformAlerts: false,
  canUseTransformAlerts: false,
  canResetTransform: false,
} as const);
```

Reply:
Updated here [`cbf8ed3` (#154007)](https://github.com/elastic/kibana/pull/154007/commits/cbf8ed3c93ad2d376f02f4481cf0daafcf1bff3d)